pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -53,6 +54,9 @@ const (
 	DefaultCNIDir     = "/opt/cni/bin"
 
 	sysctlBridgeCallIptables = "net/bridge/bridge-nf-call-iptables"
+
+	// private mac prefix safe to use
```

Comment:
Is this considered a bugfix or a feature ? Should we use the alpha
gate/feature flags for this one too ?


On Fri, Aug 19, 2016 at 5:05 PM, Minhan Xia <notifications@github.com>
wrote:

> In pkg/kubelet/network/kubenet/kubenet_linux.go
> <https://github.com/kubernetes/kubernetes/pull/28717#discussion_r75567126>
> :
>
> > @@ -53,6 +54,9 @@ const (
> >  	DefaultCNIDir     = "/opt/cni/bin"
> >
> >  	sysctlBridgeCallIptables = "net/bridge/bridge-nf-call-iptables"
> > +
> > +	// private mac prefix safe to use
>
> There is a bit to set universally administered addresses or locally
> administered addresses.
> https://en.wikipedia.org/wiki/MAC_address
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/28717/files/c8992c0a2bcfb216c073c03ff6cc551915f99003#r75567126>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ARrr7bOfawobwMAJucrj7N1VlUwu9OgVks5qhkSsgaJpZM4JIcoW>
> .
>